build/package/Microsoft.DotNet.Cli.Installer.PKG.targets

```diff
@@ -1,10 +1,132 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    
-    <UsingTask TaskName="PkgTargets" AssemblyFile="$(CLIBuildDll)" />
+    <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(CLIBuildDll)" />
+  
+    <Target Name="SetupPkgInputsOutputs">
+      <PropertyGroup>
+        <InstallerOutputDirectory>$(PackagesDirectory)</InstallerOutputDirectory>
+        <PkgIntermediateDirectory>$(IntermediateDirectory)/pkgs/$(SdkVersion)</PkgIntermediateDirectory>
 
-    <Target Name="GeneratePkgs" >
-        <PkgTargets />
+        <SdkInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionSdk)$(InstallerExtension)</SdkInstallerFile>
+        <CombinedFrameworkSdkHostInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)$(BundleExtension)</CombinedFrameworkSdkHostInstallerFile>
+
+        <!-- Rename is intentional to reduce confusion over generated vs downloaded pkg's -->
+        <DownloadedSharedFrameworkInstallerFile>$(SharedFrameworkInstallerFile)</DownloadedSharedFrameworkInstallerFile> 
+        <DownloadedHostFxrInstallerFile>$(HostFxrInstallerFile)</DownloadedHostFxrInstallerFile>
+        <DownloadedSharedHostInstallerFile>$(SharedHostInstallerFile)</DownloadedSharedHostInstallerFile>
+        
+        <!-- Properties for pkg build -->
+        <SharedHostComponentId>com.microsoft.dotnet.sharedhost.component.osx.x64</SharedHostComponentId>
+        <HostFxrComponentId>com.microsoft.dotnet.hostfxr.component.osx.x64</HostFxrComponentId>
+        <SharedFrameworkComponentId>com.microsoft.dotnet.sharedframework.$(SharedFrameworkNugetName).$(SharedFrameworkVersion).component.osx.x64</SharedFrameworkComponentId>
+        <SdkComponentId>com.microsoft.dotnet.dev.$(SdkVersion).component.osx.x64</SdkComponentId>
+        <SdkProductArchiveId>com.microsoft.dotnet.dev.$(SdkVersion).osx.x64</SdkProductArchiveId>
+        
+        <PkgInstallDirectory>/usr/local/share/dotnet</PkgInstallDirectory>
+
+        <SdkPkgScriptsDirectory>$(RepoRoot)/packaging/osx/clisdk/scripts</SdkPkgScriptsDirectory>
```

Comment:
We can make a property for `$(RepoRoot)/packaging/osx/clisdk` and not have it duped so many times below.